gettsim/soz_vers.py

```diff
@@ -171,55 +198,90 @@ def calc_midi_contributions(person, params):
 
 
 def calc_midi_f(params):
-    """ I have no idea what this function calculates. What means f?"""
-    an_anteil = params["grvbs"] + params["gpvbs"] + params["alvbs"] + params["gkvbs_an"]
-    ag_anteil = params["grvbs"] + params["gpvbs"] + params["alvbs"] + params["gkvbs_ag"]
+
+    # Todo: I have no idea what this function calculates. What means f?
+
+    an_anteil = (
+        params["soz_vers_beitr"]["rentenv"]
+        + params["soz_vers_beitr"]["pflegev"]["standard"]
+        + params["soz_vers_beitr"]["arbeitsl_v"]
+        + params["soz_vers_beitr"]["ges_krankv"]["an"]
+    )
+    ag_anteil = (
+        params["soz_vers_beitr"]["rentenv"]
+        + params["soz_vers_beitr"]["pflegev"]["standard"]
+        + params["soz_vers_beitr"]["arbeitsl_v"]
+        + params["soz_vers_beitr"]["ges_krankv"]["ag"]
+    )
     dbsv = an_anteil + ag_anteil
-    pauschmini = params["mini_ag_gkv"] + params["mini_ag_grv"] + params["stpag"]
+    pauschmini = (
+        params["ag_abgaben_geringf"]["ges_krankenv"]
+        + params["ag_abgaben_geringf"]["rentenv"]
+        + params["ag_abgaben_geringf"]["st"]
+    )
     f = round(pauschmini / dbsv, 4)
     return f
 
 
 def calc_midi_bemessungsentgelt(person, params):
```

Comment:
for docstring: "Implements formula from §163 (10) SGB VI"

Reply:
Added